packages/material-ui/src/CssBaseline/CssBaseline.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable react/no-unused-prop-types */
```

Comment:
It seems we can kil this line.

Reply:
Maybe it was used for ` classes: PropTypes.object.isRequired,`. We don't use this prop either in the component. I would kill it too.